coin/exchange/kraken.py

```diff
@@ -80,7 +100,7 @@ def _parse_result(self, data):
     asset = data[self.asset_pair]
     currency = [item['currency'] for item in CONFIG['asset_pairs'] if item['code'] == self.asset_pair][0]
 
-    label = currency + utils.decimal_round(asset['c'][0])
+    label = currency + asset['c'][0]
```

Comment:
See above: For XLM for example, which is currently 0.038.. EUR, the rounding does not really make sense. For the current price, rounding at this point means rounding away a bit more than 10% of the currency value.